### ISSUE TITLE
Proposal: Where to Buy 7kaa and 7k2

### ISSUE THREAD

**Issue by W-Damian** [lennyhans/7kaa-static-site#9]
I propose having a link for buying Seven Kingdoms. 7k2 HD should definitely be highlighted on the static site, but I also propose linking to other forms of supporting Enlight such as linking to Steam, GoG, and Retroism. I know that some Steam users buy 7kaa and incorporate it into Steam for added benefits. As for buying 7kaa, I can do a small write up about the differences.

**Comment by lennyhans** [comment]
Yeah, that seems fair, I think "Get the game" or "Get Seven Kingdoms" will work for the download/buy links
